src/main/java/com/github/hokkaydo/eplbot/module/asciinema/AsciinemaListener.java

```diff
@@ -0,0 +1,76 @@
+package com.github.hokkaydo.eplbot.module.asciinema;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.utils.AttachmentProxy;
+import net.dv8tion.jda.api.utils.FileUpload;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.function.Consumer;
+
+public class AsciinemaListener extends ListenerAdapter {
+
+    private final Long guildId;
+    AsciinemaListener(Long guildId) {
+        this.guildId = guildId;
+    }
+    private static final Random random = new Random();
+
+
+    @Override
+    public void onMessageReceived(@NotNull MessageReceivedEvent event) {
+        List<Message.Attachment> attachments = event.getMessage().getAttachments();
+        if (attachments.isEmpty()) return; // no attachments on the message!
+        for (Message.Attachment att: attachments) {
+            if (Objects.equals(att.getFileExtension(), "cast")) {
```

Comment:
Use early return

Reply:
wdym by early return ? If i find one cast file i return from the function ?

Reply:
Actually "early continue" rather
`if (!Object.equals(att.getFileExtension(), "cast")) continue;`
It helps keeping the code readable by removing one level of indentation